saga-sdk-java/src/main/java/games/mythical/saga/sdk/client/SagaItemTypeClient.java

```diff
@@ -96,8 +96,8 @@ public String createItemType(String itemTypeId,
                     .setName(name)
                     .setSymbol(symbol)
                     .setMaxSupply(maxSupply)
-                    .setRandomize(randomize)
```

Comment:
To make sure this is backward compatible, if someone passes in randomize from the SDK, just convert it to the MintMode somewhere right?   Maybe in the SDK itself?  Actually no it should be removed from the SDK so when people update it, they can't use it anymore.  Will need to go into either gateway or items?

Reply:
items checks if mintmode is unreconized and converts it to randomize if so

Reply:
What i mean is if you get an old request with randomize=true (in gateway or items or ?) then you convert it to MintMode.RANDOM somewhere